rust/kcl-lib/tests/artifact_graph_example_code1/ops.snap

```diff
@@ -51,6 +51,17 @@ description: Operations executed artifact_graph_example_code1.kcl
     }
   },
   {
+    "type": "KclStdLibCall",
```

Comment:
Just wondering, how did the order of these change? I thought we were sorting by key name, but this is much nicer.

Reply:
I have no idea. I don't think I did anything to affect this, maybe just different serialisation for stdlib vs kclstlib?